Integrations/integration-Carbon_Black_Enterprise_Response.yml

```diff
@@ -1364,8 +1401,8 @@ script:
       predefined:
       - "yes"
       - "no"
-      defaultValue: "yes"
       description: Whether to decompress results
```

Comment:
add deprecated to the command in general

Reply:
?

Reply:
done